apps/nameai.dev/app/tokenization/page.tsx

```diff
@@ -34,3 +40,32 @@ export default async function TokenizePage(props: TokenizePageProps) {
     </div>
   );
 }
+
+export async function generateMetadata(props: Props): Promise<Metadata> {
+  const searchParams = await props.searchParams;
+  const label = searchParams?.label || "";
+
+  if (!label) return notFound();
```

Comment:
This is actually incorrect. I will change the behaviour here so that the page has different metadata IF there is a label in the query string, otherwise, show a fallback value.